modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -15,9 +15,40 @@ The following steps show how to integrate Service Mesh with user-workload monito
 
 * User-workload monitoring is enabled.
 * {SMProductName} Operator 2.4 is installed.
+* Kiali Operator 1.65 is installed.
 
 .Procedure
 
+. Configure Kiali for `user-workload-monitoring`:
++
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali-user-workload-monitoring
+  namespace: istio-system-1
+spec:
+  external_services:
+    istio:
+      config_map_name: istio-basic
+      istio_sidecar_injector_config_map_name: istio-sidecar-injector-basic
+      istiod_deployment_name: istiod-basic
+      istiod_pod_monitoring_port: 15014
+      url_service_version: 'http://istiod-basic.istio-system-1:15014/version'
+    prometheus:
+      auth:
+        token: secret:thanos-querier-web-token:token
+        type: bearer
+        use_kiali_token: false
+      query_scope:
+        mesh_id: "basic_istio-system-1"
```

Comment:
```suggestion
        mesh_id: "basic_istio-system"
```
Btw. now I see that we should change `replacement: "<smcp_name>-<smcp_namespace>" # <2>` in Pod/Service monitors and explain in the <2> that this value could be anything unique, e.g. `<smcp_name>-<smcp_namespace>`.